posts/wordpress-to-jamstack.md

```diff
@@ -0,0 +1,152 @@
+---
+authors:
+  - Brandon Landfried
+tags:
+  - Javascript
+  - NextJS
+  - React
+  - JAMstack
+date: 2021-04-06T00:00:00.000Z
+title: "Wordpress to JAMstack"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2021/01/stargate-main.png
+---
+
+I was approached a few months into the pandemic by my parents church asking if I would be able to update their website to make it modern. With everyone in lock down and having restrictions on gatherings the church had started to rely more on live streams and posting their sermons on the website. This means their website was getting more traffic and it was being brought up by members that it was not a very good experience.
+
+In this blog post I will walk through the steps I took and some of my decision making on converting the outdated [Wordpress](https://wordpress.com/) website to a [JAMstack](https://jamstack.org/) powered website that is modern, responsive, and best of all free!
+
+Since this was something I was doing as a favor I took the opportunity to make it a learning experience and use some services and frameworks that were newer to me and have fun with it.
+
+# Technologies Used
+
+- [NextJS](https://nextjs.org/)
+- [TinaCMS](https://tina.io/)
+- [Vercel](https://vercel.com/)
+- [DataStax Astra](https://www.datastax.com/)
+- [SendGrid](https://sendgrid.com/)
+- [EasyCron](https://www.easycron.com/)
```

Comment:
It would be nice to have a little blurb alongside each technology to talk about what they do conceptually.

Reply:
*Next.js